### ISSUE TITLE
Can we remove empty tags from Xml

### ISSUE THREAD

**Issue by SindhuKonda** [Tickaroo/tikxml#106]
Can we remove empty tags/elements from xml  which was generated using tik xml? If yes , can you please mention sample?

**Comment by SindhuKonda** [comment]
Can we do it in type adapter ?

**Comment by sockeqwe** [comment]
At the moment there is no support for this, but its not that hard to add
support for it but must be added in the annotation processor

SindhuKonda <notifications@github.com> schrieb am Mi., 26. Sep. 2018, 12:24:

**Comment by sockeqwe** [comment]
Feel free to send a Pull Request to fix this ...

**Comment by SindhuKonda** [comment]
Pull request ?

**Comment by sockeqwe** [comment]
Yeah, in case that this is very urgent for you, and you would like to contribute to the tikxml project, feel free to add this functionality and contribute back your changes (this is called Pull Request on Github).

**Comment by WeaponMan** [comment]
Looks like tree problem.. You will check every property and the element itself with null checks before writing the beginning of that element, with one exception that is root element has to be written always.

If I thinking right this could go hand in hand with #57.
